server/db/__mocks__/pg.js

```diff
@@ -0,0 +1,7 @@
+const pg = jest.genMockFromModule('pg');
+
+const query = (sql, params) => new Promise(resolve => resolve({ query: sql, params }));
+
+pg.Client.prototype.query = query;
+
```

Comment:
Is 'pg' referring to postgres?

Reply:
yup!  pg is the name of the node-postgres module.  Here I am basically replacing the real pg.Cient.query() function with my fake mock that just returns what was passed in so I can check what my db module sends in for my tests